test/trackler/problem_test.rb

```diff
@@ -4,25 +4,65 @@
 class ProblemTest < Minitest::Test
   def test_existing_problem
     problem = Trackler::Problem.new('hello-world', FIXTURE_PATH)
-
     assert problem.exists?, "hello world files not found in the metadata dir"
+  end
+
+  def test_slug
```

Comment:
I wish Ruby had the equivalent of t.Errorf as well as t.Fatalf in the Go testing package so we could stick all the trivial assertions in the same test. *sigh* Nothing to change here, I'm just wishing out loud that we didn't have to spread silly things out across 50 lines of code when 10 would be more readable.